deploy/mutatingwebhook.yaml

```diff
@@ -13,6 +13,11 @@ webhooks:
       name: pod-identity-webhook
       namespace: default
       path: "/mutate"
+  objectSelector:
+    matchExpressions:
+      - key: eks.amazonaws.com/skip-identity-webhook
```

Comment:
I am fine with this change, would love to hear @micahhausler opinion.

Reply:
@micahhausler could you take a look at this?

Reply:
@prateekgogia Can I have you re-review this?

Reply:
Can we change this to “eks.amazonaws.com/skip-pod-identity-webhook”. Identity webhook may be too generic if EKS adds more identity related webhooks in the future.

Reply:
@DanielCKennedy makes sense! Fixed!